src/modules/commands/common/membercount.ts

```diff
@@ -0,0 +1,19 @@
+import { Command } from '@commands/CommandHandler'
+import { censusApi } from '@planetside/CensusApi'
+import { constants } from '@app/global/constants'
+
+export default new Command({
+  keyword: 'membercount',
+  description: 'count the members of the outfit',
+  help: 'Usage: `{prefix}membercount` - count the members of the outfit',
+  callback: async ({ args, reply }) => {
+    if (args.length === 0) return
```

Comment:
0 args should show the help message, check e.g. diceroll for reference.

Reply:
Sorry, I take it back. Since there are no arguments (at least not yet), this should be
```ts
if (args.length > 0) return
```
(so still needs fixing)

Reply:
Done

Reply:
Not done.